README.md

```diff
@@ -1,16 +1,17 @@
 # Percona Monitoring and Management 2.x
```

Comment:
I think it's time to name it just like this:
```suggestion
# Percona Monitoring and Management
```

Reply:
PMM 1.x is gone :)